### PR TITLE
[context.tags] 0.0.4

### PR DESCRIPTION
**Pull request by zosky**
### Description

this puts a Tags item under context > manage (for movies and tvShows) ... selecting it will bring up a multiSelect with all existing tags. it will pre-select tags already applied to the current-item... allowing the user to select more or un-select and save ([ok] or [cancel]). 

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

### CODE REVIEW

File: context.tags/addon.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    title = xbmcaddon.Addon().getLocalizedString(32001)
+    returned = dialog.multiselect(title, tags, preselect=preSelectedTAGS)
+    # IF NOT CANCELED
+    if returned is not None:
+        selected = []
+        for n in returned:
+            selected.append(tags[n])
+
+        # SUBMIT
+        SETnewTAGS = {'jsonrpc': '2.0', 'id': 'tagSelector',
```
Comment (Razzeee) [inline context.tags/addon.py]: Just to be clear this won't remove tags that you deselected?

Comment (zosky) [inline context.tags/addon.py]: i'm not explicitly telling it to remove tags that are deselected... it just happens as part of how kodi handles this method ? 

check this, i started with a movie that had 1 tag 
- deselect it ( send _tag: [ ]_ )
```
02:54:35.198 T:140539475240704   DEBUG: Mysql execute: DELETE FROM tag_link WHERE media_id=1633 AND media_type='movie'
```

- select 3 ( send _tag: ["me", "wife", "kid"]_
```
02:56:09.509 T:140541695956736   DEBUG: Mysql execute: DELETE FROM tag_link WHERE media_id=1633 AND media_type='movie'
02:56:09.514 T:140541695956736   DEBUG: Mysql execute: INSERT INTO tag_link (tag_id,media_id,media_type) VALUES(1,1633,'movie')
02:56:09.536 T:140541695956736   DEBUG: Mysql execute: INSERT INTO tag_link (tag_id,media_id,media_type) VALUES(2,1633,'movie')
02:56:09.540 T:140541695956736   DEBUG: Mysql execute: INSERT INTO tag_link (tag_id,media_id,media_type) VALUES(4,1633,'movie')
```

- deselect tag 2 & 4 ( send _tag: ["me"]_)
```
02:56:57.668 T:140540173838080   DEBUG: Mysql execute: DELETE FROM tag_link WHERE media_id=1633 AND media_type='movie'
02:56:57.669 T:140540173838080   DEBUG: Mysql execute: INSERT INTO tag_link (tag_id,media_id,media_type) VALUES(1,1633,'movie')
```

Comment (Razzeee) [inline context.tags/addon.py]: Interesting, didn't know that :)

File: context.tags/addon.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <menu id="kodi.core.manage">
+      <item library="addon.py">
+        <label>32000</label>
+        <visible>String.IsEqual(ListItem.dbtype,movie) | String.IsEqual(ListItem.dbtype,tvshow)</visible>
+      </item>
+    </menu>
+  </extension>
+  <extension point="kodi.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">Manage tags for the selected item from the context menu</summary>
```
Comment (enen92) [inline context.tags/addon.xml]: please change those "en" attributes to "en_GB", it is the new default in krypton. In fact you are already using it in the languages folder :)

Comment (zosky) [inline context.tags/addon.xml]: sorted. thanks for the catch.

**Comment by enen92** [comment]
Thanks for adjusting

**Pull request merged**